tests/main/lxd-mount-units/task.yaml

```diff
@@ -30,13 +30,8 @@ execute: |
         lxd.lxc config set core.proxy_https "$http_proxy"
     fi
 
-    # There isn't an image for kinetic yet, let's just check that lxd snap can be installed
-    if os.query is-ubuntu 22.10; then
-        exit
-    fi
-
-    VERSION_ID="$(. /etc/os-release && echo "$VERSION_ID" )"
-    lxd.lxc launch --quiet "ubuntu:$VERSION_ID" ubuntu
```

Comment:
I'm not sure we should change this for the existing images. The `ubuntu: ` prefix in lxc indicates that it's an "official" ubuntu image. So when we have them we should use those. The `images:` prefix is the community maintained images. They are slightly different and have the latest dev release too but they are not comming from Canonical.

Reply:
Ok, makes sense, in that case I'll update that just for kinetic